ruma-api-macros/src/api/attribute.rs

```diff
@@ -24,10 +24,6 @@ pub enum Meta {
 
 impl Meta {
     /// Check if the given attribute is a ruma_api attribute. If it is, parse it.
-    ///
-    /// # Panics
-    ///
-    /// Panics if the given attribute is a ruma_api attribute, but fails to parse.
```

Comment:
Huh

Reply:
The method that the doc belongs to doesn't panic, that I could find at least? it just calls `.is_ident` and `.parse_args` and neither of them panic.

Reply:
Yeah, the "Huh" was my reaction to that doc comment previously talking about panics :D